internal/dataplane/kongstate/customentity.go

```diff
@@ -89,3 +98,53 @@ func ExtractEntityFieldDefinitions(schema kong.Schema) EntitySchema {
 	}
 	return retSchema
 }
+
+// KnownEntityTypeList includes the types of standard Kong entities that is processed elsewhere in KIC.
+// So the entities cannot be specified via KongCustomEntity types.
+var KnownEntityTypeList = []string{
+	"services",
+	"routes",
+	"upstreams",
+	"targets",
+	"consumers",
+	"consumer_groups",
+	"plugins",
+}
+
+// KnownEntityTypeMap is the map including standard entities used for checking whether a custom entity has one of the types.
+var KnownEntityTypeMap = lo.SliceToMap(KnownEntityTypeList, func(s string) (string, struct{}) { return s, struct{}{} })
```

Comment:
```suggestion
func IsKnownEntityType(entityType string) bool {
	// Types of standard Kong entities that is processed elsewhere in KIC.
	// So the entities cannot be specified via KongCustomEntity types.
	knownEntities := map[string]struct{}{
		"services":        {},
		"routes":          {},
		"upstreams":       {},
		"targets":         {},
		"consumers":       {},
		"consumer_groups": {},
		"plugins":         {},
	}
	_, ok := knownEntities[entityType]
	return ok
}
```
I think that something like above is easier to follow and use later, like in the `internal/dataplane/kongstate/kongstate.go:710`

Reply:
Done.